demos/toolbar/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2016 Google Inc. All rights reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      https://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License
+-->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>MDC Toolbar Demo</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <style>
+      .demo-container {
+        height: 500px;
+        margin: 0px;
+        font-size: 16px;
+        overflow: scroll;
+      }
+    </style>
+    <script src="../assets/material-components-web.css.js" charset="utf-8"></script>
+  </head>
+  <body class="mdc-typography">
+    <main>
+      <h2>Normal Toolbar <button type="button" onclick="document.querySelectorAll('iframe')[0].contentWindow.document.querySelector('main').hasAttribute('dir') ?document.querySelectorAll('iframe')[0].contentWindow.document.querySelector('main').removeAttribute('dir') : document.querySelectorAll('iframe')[0].contentWindow.document.querySelector('main').setAttribute('dir', 'rtl');">Toggle RTL</button></h2>
```

Comment:
I ended it up put it in a function but not yet using a `addEventListener()` just to keep things simple. If you think that is indeed needed, lmk.